
Fix possible buffer overrun with the rsyncable patch - pulled from Fedora Core.

  - Rene Rebe <reneÂ@exactcode.de>

--- gzip-1.3.3/gzip.c.orig	Thu Jan 30 21:19:36 2003
+++ gzip-1.3.3/gzip.c	Thu Jan 30 21:09:52 2003
@@ -198,7 +198,7 @@
 DECLARE(uch, inbuf,  INBUFSIZ +INBUF_EXTRA);
 DECLARE(uch, outbuf, OUTBUFSIZ+OUTBUF_EXTRA);
 DECLARE(ush, d_buf,  DIST_BUFSIZE);
-DECLARE(uch, window, 2L*WSIZE);
+DECLARE(uch, window, 2L*WSIZE + 4096);
 #ifndef MAXSEG_64K
     DECLARE(ush, tab_prefix, 1L<<BITS);
 #else
